src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -458,14 +477,8 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     this.listenToQueryStateChange = true;
   }
 
-  public async executeQuery() {
```

Comment:
Is the `executeQuery` method defined somewhere else to avoid a breaking change?

Reply:
It's not and not publicly documented. We could have some noop methods in for those before merging